[**@ethereum-multicall/utils v1.0.0**](../README.md) • **Docs**

***

[Documentation v1.0.0](../../../packages.md) / [@ethereum-multicall/utils](../README.md) / wrapError

# Function: wrapError()

> **wrapError**(`error`, `code`, `context`?): [`MulticallError`](../classes/MulticallError.md)

Helper function to wrap any error as a MulticallError

## Parameters

• **error**: `unknown`

• **code**: [`ErrorCodes`](../enumerations/ErrorCodes.md)

• **context?**: [`MulticallErrorContext`](../type-aliases/MulticallErrorContext.md)

## Returns

[`MulticallError`](../classes/MulticallError.md)

## Defined in

[packages/utils/src/errors/multicall-error.ts:129](https://github.com/niZmosis/ethereum-multicall/blob/2a2d077a99c23b464a4e40dd6375d06ce98594bd/packages/utils/src/errors/multicall-error.ts#L129)
